string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,28 @@
+/* exported firstChars */
+function firstChars(length, string) {
+  var output = '';
+  for (var i = 0; i < length; i++) {
+    if (length <= string.length) {
+      output += string[i];
+    } else if (string === '') {
+      output += '';
+    } else if (length > string.length) {
+      length = string.length;
+      output += string[i];
+    }
```

Comment:
You should probably do these checks before the loop begins, they each either the need for the loop or augment its behavior, both of which should be done before the loop starts.